frontend/ui/size.lua

```diff
@@ -90,9 +90,9 @@ local Size = {
         vertical_large = Screen:scaleBySize(5),
     },
     max_items_normalized = {
-        info_list = sizeMaxItemsNormalized(Screen:getHeight(), Device.display_dpi, 20),
-        mosaic_h = sizeMaxItemsNormalized(Screen:getHeight(), Device.display_dpi, 75),
-        mosaic_w = sizeMaxItemsNormalized(Screen:getWidth(), Device.display_dpi, 75),
+        info_list = maxItems(Screen:getHeight(), Device.display_dpi, 20),
+        mosaic_h = maxItems(Screen:getHeight(), Device.display_dpi, 75),
+        mosaic_w = maxItems(Screen:getWidth(), Device.display_dpi, 75),
```

Comment:
`Screen:getDPI` ?

(This *will* return the custom DPI value, if any, which... actually sounds desirable anyway?).

Reply:
Tested it, `Screen:getDPI()` return the one from Top menu - screen settings. This is not the same as hardware `display_dpi` defined in the `Device:init`

Reply:
My point was to use hardware `display_dpi` for this calculation.

Reply:
And what's the problem with honoring the DPI override here?

Reply:
> My point was to use hardware `display_dpi` for this calculation.

But doesn't really make sense. :-) The DPI setting overrides the "true" DPI with how large the user would like things.

Reply:
DPI override I see more like a font and UI elements scale. What if users want, say, 30 lines, but with large font?

Also, DPI override might be larger or smaller than hardware dpi. I use 120, for example, because anything higher makes menus unusably huge.

Reply:
> DPI override I see more like a font and UI elements scale. What if users want, say, 30 lines, but with large font?

That's not an argument in favor of using `display_dpi` for this calculation. That's an argument in favor of being able to set different values somehow.

> Also, DPI override might be larger or smaller than hardware dpi.

But that's the point. We don't want the hardware DPI. We only ever want that DPI.

> I use 120, for example, because anything higher makes menus unusably huge.

This feature/problem comes from `scaleBySize()` more so than the DPI. Though in the context of this PR I find that a bit of a confusing thing to say. A smaller DPI setting will make all the covers smaller too, automatically putting many more on the screen.

Reply:
300 psi override on Scribe is huge!
So, it seems it works the other way?

Reply:
> We don't want the hardware DPI. We only ever want that DPI.

In that case my chosen points (pt) for physical dimensions won't make any sense, so this PR will be meaningless if I use it with DPI override. 

I mean, 20 pt for Infolists that is currently in the PR, is 0.7 cm. With override, this choice doesn't matter anymore - it will larger/smaller.

Reply:
Yes, we want it to be larger or smaller. We don't want a value that tries to always be 0.7 cm.

It being larger/smaller means it will be properly sized by some approximation, while 0.7 cm will be too large or too small.

Reply:
Mind, you can think in terms of 0.7cm perfectly fine.

0.7 cm at 160 DPI is (.7 × 160) ÷ 2.54 = 44 px

So you set 44px.

This will also be roughly .7cm at 300 DPI. No problem there.

Where it gets a little "messy" is what `scaleBySize()` does, namely to size things up more depending on screen size in addition to DPI. This works fairly well all the way from phone size to ereader size to monitor to TV. That's mostly a matter of viewing distance. If you're looking at the screen from 2 meters you don't want that thing to be .7cm, you want it to be 2.1cm.

The correct version of this PR would be to rip out `scaleBySize()` imho.

But perhaps we could just make it user settable? Then we could also put "this device is intended to be held close" in our device definitions for example.

Reply:
Though to be clear I don't really understand what the problem is with just setting a lower DPI if you like things smaller.

Reply:
Yes, I did set it to lower. 

The problem this PR tries to solve is the hardcoded maximum number of items per page, which do not scale with DPI override.

But instead of just bumping the hardcoded value, I am trying to use a calculated value that physically makes sense.

It is not even about enforcing default values. It is about allowing to have more items if you need. 

So the larger the screen (physically) more items will fit (of the same physical size).